pkgs/servers/monitoring/prometheus/apcupsd-exporter.nix

```
@@ -1,27 +1,23 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "apcupsd-exporter";
-  version = "unstable-2019-03-14";
-
-  goPackagePath = "github.com/mdlayher/apcupsd_exporter";
-
-  goDeps = ./apcupsd-exporter_deps.nix;
+  version = "0.2.0";
 
   src = fetchFromGitHub {
     owner = "mdlayher";
     repo = "apcupsd_exporter";
-    rev = "cbd49be";
-    sha256 = "1h5z295m9bddch5bc8fppn02b31h370yns6026a1d4ygfy3w46y0";
+    rev = "v${version}";
+    sha256 = "0gjj23qdjs7rqimq95rbfw43m4l6g73j840svxjlmpd1vzzz2v2q";
   };
 
-  doCheck = true;
+  vendorSha256 = "09x8y8pmgfn897hvnk122ry460y12b8a7y5fafri5wn9vxab9r82";
 
   meta = with stdenv.lib; {
     description = "Provides a Prometheus exporter for the apcupsd Network Information Server (NIS)";
     homepage = "https://github.com/mdlayher/apcupsd_exporter";
     license = licenses.mit;
-    maintainers = with maintainers; [ maintainers."1000101" ];
+    maintainers = with maintainers; [ "1000101" mdlayher ];
```

I think we need to keep it as this: 

```suggestion
    maintainers = with maintainers; [ maintainers."1000101" mdlayher ];
```

---

Ack, sorry!

---

```
nix-repl> builtins.toJSON (with stdenv.lib; with maintainers;  [ maintainers."1000101" mdlayher ])
"[{\"email\":\"b1000101@pm.me\",\"github\":\"1000101\",\"githubId\":791309,\"name\":\"Jan Hrnko\"},{\"email\":\"mdlayher@gmail.com\",\"github\":\"mdlayher\",\"githubId\":1926905,\"name\":\"Matt Layher\"}]"

nix-repl> builtins.toJSON (with stdenv.lib; with maintainers;  [ "1000101" mdlayher ])
"[\"1000101\",{\"email\":\"mdlayher@gmail.com\",\"github\":\"mdlayher\",\"githubId\":1926905,\"name\":\"Matt Layher\"}]"
```